x-pack/legacy/plugins/siem/public/components/ml/anomalies_table_data.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import chrome from 'ui/chrome';
+import { Anomalies, InfluencerInput } from './types';
+
+export interface Body {
+  jobIds: string[];
+  criteriaFields: string[];
+  influencers: InfluencerInput[];
+  aggregationInterval: string;
+  threshold: number;
+  earliestMs: number;
+  latestMs: number;
+  dateFormatTz: string;
+  maxRecords: number;
+  maxExamples: number;
+}
+
+export const anomaliesTableData = async (
```

Comment:
May want to throw this file and any other corresponding api calls into an `api` subdir so they're easier to grok.

Reply:
Made a folder structure with:

* api
* anomaly
* influencers
* links
* score
* tables

to add some grouping. I put _just_ the REST API inside of the `api` folder.